test/sequential/xyk-SDK.nonce.test.ts

```diff
@@ -185,50 +187,48 @@ describe.skip("SDK test - Nonce tests - user", () => {
     let eventResult = getEventResultFromMangataTx(events);
     expect(eventResult.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
     //perform an operation without custom nonce.
-    const events2 = await mangata.sellAsset(
+    const events2 = await sellAsset(
       testUser.keyRingPair,
-      firstCurrency.toString(),
-      MGA_ASSET_ID.toString(),
+      firstCurrency,
+      MGA_ASSET_ID,
       new BN(1000),
       new BN(0)
     );
     eventResult = getEventResultFromMangataTx(events2);
     expect(eventResult.state).toEqual(ExtrinsicResult.ExtrinsicSuccess);
 
-    userNonce.push(await mangata.getNonce(testUser.keyRingPair.address));
+    userNonce.push(await mangata.query.getNonce(testUser.keyRingPair.address));
     expect(parseFloat(userNonce[1].toString())).toBeGreaterThan(
       parseFloat(userNonce[0].toString())
     );
   });
   test.skip("[BUG?] SDK- Nonce management - Transaction outdated", async () => {
     const userNonce = [];
     // perform an operation to have nonce > 0.
-    await mangata.sellAsset(
+    await sellAsset(
       testUser.keyRingPair,
```

Comment:
Isn't it better to use `User `instead of `KeyringPair `here and in similar functions?

Reply:
SDK does not know about keyringpair, and inside sellAsset, it seems we are not using the User object for anything else.  I think we should keep using User, but I remember having some issues when migrating somehow.